.github/workflows/build_steps.yml

```
@@ -55,6 +55,9 @@ jobs:
       ARCTICDB_BUILD_DIR: ${{matrix.build_dir}}
       ARCTICDB_VCPKG_INSTALLED_DIR: ${{matrix.vcpkg_installed_dir}}
       ARCTICDB_VCPKG_PACKAGES_DIR: ${{matrix.vcpkg_packages_dir}}
+      MACOSX_DEPLOYMENT_TARGET: "11.0"
+      CMAKE_OSX_DEPLOYMENT_TARGET: "11.0"
```

Seems to be required by the three way comparison for lexicographic ordering.

```suggestion
      MACOSX_DEPLOYMENT_TARGET: "12.0"
      CMAKE_OSX_DEPLOYMENT_TARGET: "12.0"
```